foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -56,6 +56,10 @@ An important implication to notice here is that when you specify `flex-grow` or
 
 `flex-basis` simply sets the initial size of a flex item, so any sort of `flex-grow`ing or `flex-shrink`ing starts from that baseline size. The shorthand value defaults to `flex-basis: 0%`. The reason we had to change it to `auto` in the `flex-shrink` example is that with the basis set to `0`, those items would ignore the item's width, and everything would shrink evenly. Using `auto` as a flex-basis tells the item to check for a width declaration (`width: 250px`).
 
+### flex-auto
+
+`flex: auto` is one of the short hands of flex. When `flex: auto` is called it inherits the values of `flex-grow: 1 flex-shrink: 1 flex:basis: auto` or `flex: 1 1 auto` using the flex shorthand. Note that `flex: auto` is not the default value when using the flex short hand despite the name being "auto" which may be slightly confusing at first. The default value for flex is `flex: 0 1 auto` or `flex: initial` which we won't get too in depth about here.
```

Comment:
**thought:** What do you think about including one more line in the section that could summarize the standard use case of `flex: auto`?

If you look at the docs from w3 or MDN they both give a brief summary of how the `flex: auto` property will size the item by its `width` and `height` properties but still flex to the free space of the container. I think we could either add our own line here that doesn't directly copy w3/MDN, or use an exact quote and give source attribution to one of those docs. 

Let me know your thoughts on adding something like this to the section. Or if you think that will be too much here, we could also tell the learner something like "you will learn more about when to use `flex: auto` when reading through the docs in the Assignment section."

Reply:
Going to be working on this tommorow! I will re-request a review upon finishing

Reply:
Hey, Upon looking at this i'm not sure if this aligns with our purpose of informing about flex-auto. If we leave the note header as  "What is flex-auto" as a continuation explanation for the flex-basis article, Our goal would be to clarify about the definition of flex-auto and what it is in itself rather than how to use it and it's potential use cases.

I've added a line equivalent to the  

> you will learn more about when to use flex: auto when reading through the docs in the Assignment section."

Described earlier, Let me know your viewpoint on this.